pom.xml

```diff
@@ -102,7 +102,8 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>branch-api</artifactId>
-            <version>2.0.2</version>
+            <!-- TODO: Remove when branch-api is released -->
+            <version>2.0.5-20170213.115501-2</version>
```

Comment:
Is this downstream of some other PR? If so, mention it in this PR’s description, and in this comment.